classes/03class/exercises/c03-docker04/deniseddi/c03-docker04-01.txt

```diff
@@ -0,0 +1,15 @@
+1. Commands used to clean up containers and imagesfrom my local (I've used them a bit during the previous docker exercises):
+
+•   For each container I first stopped, then deleted like so:
+denisegomes$ docker stop <container ID>
+denisegomes$ docker rm <container ID>
+
+•	For images:
+denisegomes$ docker rmi <image ID>
```

Comment:
take a look in the prune option of the `docker image` and `docker container` commands..
It might be useful in the future.